package.json

```diff
@@ -19,11 +19,13 @@
     "@ionic/eslint-config": "^0.3.0",
     "@ionic/prettier-config": "~1.0.1",
     "@ionic/swiftlint-config": "^1.1.2",
+    "@types/tar": "^4.0.4",
     "eslint": "^7.11.0",
     "husky": "^5.0.4",
     "lerna": "^3.22.1",
     "prettier": "~2.2.0",
     "prettier-plugin-java": "~1.0.0",
-    "swiftlint": "^1.0.1"
+    "swiftlint": "^1.0.1",
+    "tar": "^6.0.5"
```

Comment:
should this and the types/tar packages be here on the root? they are already on the cli's package.json

Reply:
Technically yes because `scripts/pack-cli-assets.mjs` also imports tar.

Reply:
ah, feels weird to have the script outside the cli folder, but considering all the other code shared it's probably ok